### PR TITLE
This PR adds SPC integration

### PR DESCRIPTION
**Pull request by yathishram**
Completed 
1. Adding all SPC contract functions
2. Adding a documentSigning functionality in web3-contract.js in utils folder
3. Adding deployment functionality for SPC contract
4. Adds SPC deployment Test

Pending
1. Rest of the getter functions of SPC which will be added as required for later use. 

SPC deployment demo
https://www.loom.com/share/4526d40554b5462cbd2bbbbd0920a961

### CODE REVIEW

**Comment by yathishram** [comment]
https://www.loom.com/share/744a5b7212ab4dd6bf35c3c9326a0eed

File: src/lib/contracts/DeployWorkflow.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { addProcedureContract, addAgreementContract } from './MasterContract';
+
+const Agreement_ContractAbi = require('../../build/ArbitrationAgreement_abi.json');
+const SPC_ContractAbi = require('../../build/SPC_abi.json');
+
+const Web3 = require('web3');
+const Agreement_ContractBin = require('../../build/ArbitrationAgreement_bin.json').binary;
+const SPC_ContractBin = require('../../build/SPC_bin.json').binary;
+
+const Agreement_ContractReceipt = {
```
Comment (koshikraj) [inline src/lib/contracts/DeployWorkflow.js]: Remove this constant if not used.

File: src/components/Login/SignUp.js [hunk 2]
```diff
@@ -5,22 +5,23 @@ import { useHistory } from 'react-router-dom';
...
+import { Web3Contract } from '../../utils/web3-contracts';
+import { createUser, userMap } from '../../lib/contracts/MasterContract';
 import 'antd/dist/antd.css';
 import './SingnUp.Style.css';
 
 const { Step } = Steps;
 const { Option, OptGroup } = Select;
 
-const accounts = require('../../wallet/keys.js');
+const Accounts = require('../../wallet/keys.js');
```
Comment (koshikraj) [inline src/components/Login/SignUp.js]: Are we using this `Accounts` now since we are creating the user wallet during the signup?

**Comment by koshikraj** [comment]
LGTM. @Koushith can you review the changes after the merge?

File: src/components/Login/SignUp.js (hunk 2 above)
Comment (yathishram) [reply src/components/Login/SignUp.js]: No, I changed it cause there was a small variable conflict.

**Pull request merged**